xbmc/GUIInfoManager.cpp

```diff
@@ -3925,6 +3925,13 @@ const infomap musicplayer[] =    {{ "title",            MUSICPLAYER_TITLE },
 ///     @skinning_v20 **[New Infolabel]** \link VideoPlayer_HdrType `VideoPlayer.HdrType`\endlink
 ///     <p>
 ///   }
+///   \table_row3{   <b>`VideoPlayer.VideoVersionName`</b>,
+///                  \anchor VideoPlayer_VideoVersionName
+///                  _string_,
+///     @return String containing the name of the version of a video - empty if no version available
```

Comment:
Maybe make a bit more clear that this is returning the version for the currently playing video, if any.

Like: "String containing the name of the version of the currently playing video - empty if no version available."

Reply:
OK did this then squashed/force pushed.  Maybe I should have done a --fixup commit instead?